pkg/proc/proc_test.go

```diff
@@ -5926,6 +5927,120 @@ func TestStacktraceExtlinkMac(t *testing.T) {
 	})
 }
 
+func testGenRedirect(t *testing.T, fixture protest.Fixture, expectStdout string, expectStderr string, errChan chan error) (redirect [3]proc.OutputRedirect, cancelFunc func(), err error) {
```

Comment:
IMO this functionality is adequately tested throught the dap layer and there is no need for this test here.

Reply:
done.